### PR TITLE
Speedy: cleanup tests for builtins

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**
follow up of #7314.

CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [X] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [X] Include appropriate tests
- [X] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/svalue/EqualitySpec.scala [hunk 1]
```diff
@@ -1,470 +0,0 @@
-// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```
Comment (hurryabit) [inline daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/svalue/EqualitySpec.scala:1]: Why can we delete this whole test suite?

File: daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/svalue/OrderingSpec.scala [hunk 2]
```diff
@@ -4,10 +4,7 @@
 package com.daml.lf.speedy
 package svalue
 
-import java.util
```
Comment (hurryabit) [inline daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/svalue/OrderingSpec.scala:7]: Why can we delete all the stuff in here?

**Review by hurryabit — COMMENTED** [review]
I have a few questions.

File: daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/svalue/EqualitySpec.scala (hunk 1 above)
Comment (remyhaemmerle-da) [reply daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/svalue/EqualitySpec.scala:1]: This is tested extensively in [ComparisonSBuiltinTest](https://github.com/digital-asset/daml/blob/master/daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/ComparisonSBuiltinTest.scala)

File: daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/svalue/OrderingSpec.scala (hunk 2 above)
Comment (remyhaemmerle-da) [reply daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/svalue/OrderingSpec.scala:7]: This is tested extensively in [ComparisonSBuiltinTest](https://github.com/digital-asset/daml/blob/master/daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/ComparisonSBuiltinTest.scala)

**Review by hurryabit — APPROVED** [review]
🔥

**Pull request merged**